path_tolerance: 0.001
goal_tolerance: 0.001

robot_description_name: "robot_description"
move_group: "manipulator"
use_execution_trajectory: false


list_of_trajectories:
- temp
- home
- from_home_to_1
- from_1_to_2
- from_2_to_3
- from_3_to_4
- from_4_to_home

temp:
  positions:
  - [-0.014996830617086232, -1.919753213922018, 2.1971996466266077, -1.801255842248434, -0.04605275789369756, -1.5779064337359827]
  velocities:
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  time_from_start:
  - 0.0
  joint_names:
  - shoulder_pan_joint
  - shoulder_lift_joint
  - elbow_joint
  - wrist_1_joint
  - wrist_2_joint
  - wrist_3_joint

home:
  positions:
  - [1.570768117904663, -2.443449159661764, 2.4434359709369105, -1.5707813066295166, 1.5707578659057617, 3.814697265625e-06]
  velocities:
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  time_from_start:
  - 0.0
  joint_names:
  - shoulder_pan_joint
  - shoulder_lift_joint
  - elbow_joint
  - wrist_1_joint
  - wrist_2_joint
  - wrist_3_joint

from_home_to_1:
  positions:
  - [1.570768117904663, -2.443449159661764, 2.4434359709369105, -1.5707813066295166, 1.5707578659057617, 3.814697265625e-06]
  - [1.001410961151123, -2.337865015069479, 2.431357447301046, -1.662931581536764, 1.571694016456604, 0.5695419311523438]
  - [0.663115382194519, -1.9535099468626917, 2.2879205385791224, -1.9000221691527308, 1.558409333229065, 0.8770067095756531]
  - [0.6010869741439819, -1.6225162945189417, 2.0371931234942835, -1.970828195611471, 1.5654218196868896, 0.9389309287071228]
  - [0.5282213091850281, -1.2715742152980347, 1.6363580862628382, -1.9212967358031214, 1.564538836479187, 1.0113383531570435]
  velocities:
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  time_from_start:
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  joint_names:
  - shoulder_pan_joint
  - shoulder_lift_joint
  - elbow_joint
  - wrist_1_joint
  - wrist_2_joint
  - wrist_3_joint

from_1_to_2:
  positions:
  - [0.5282213091850281, -1.2715742152980347, 1.6363580862628382, -1.9212967358031214, 1.564538836479187, 1.0113383531570435]
  - [0.7634989619255066, -1.1257265669158478, 2.239444080983297, -2.68892826656484, 1.549798607826233, 0.7772048711776733]
  - [1.4974051713943481, -1.058800832634308, 2.597203795109884, -3.094701429406637, 1.5811653137207031, 0.05262090638279915]
  - [2.3737573623657227, -1.0404153627208252, 1.9604547659503382, -2.4731346569456996, 1.606820821762085, -0.8673213163958948]
  - [2.2175071239471436, -0.9298135799220582, 1.0707104841815394, -1.7035113773741664, 1.563995599746704, -0.6534898916827601]
  velocities:
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  time_from_start:
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  joint_names:
  - shoulder_pan_joint
  - shoulder_lift_joint
  - elbow_joint
  - wrist_1_joint
  - wrist_2_joint
  - wrist_3_joint

from_2_to_3:
  positions:
  - [2.2175071239471436, -0.9298135799220582, 1.0707104841815394, -1.7035113773741664, 1.563995599746704, -0.6534898916827601]
  - [1.7955834865570068, -1.1339561504176636, 1.2349260489093226, -1.6787940464415492, 1.5682356357574463, -0.2588260809527796]
  - [1.5125757455825806, -0.9817204040339966, 1.3219440619098108, -1.9251581631102503, 1.563894271850586, 0.05428904667496681]
  - [0.9431952238082886, -0.9670897287181397, 0.9264043013202112, -1.5154418398490925, 1.6421732902526855, 0.648362398147583]
  - [0.4054431915283203, -1.469146878724434, 1.6495121161090296, -1.7580234012999476, 1.5765190124511719, 1.1563398838043213]
  velocities:
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  time_from_start:
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  joint_names:
  - shoulder_pan_joint
  - shoulder_lift_joint
  - elbow_joint
  - wrist_1_joint
  - wrist_2_joint
  - wrist_3_joint

from_3_to_4:
  positions:
  - [0.4054431915283203, -1.469146878724434, 1.6495121161090296, -1.7580234012999476, 1.5765190124511719, 1.1563398838043213]
  - [-0.006528202687398732, -1.7020799122252406, 1.7243946234332483, -1.5961758098998011, 1.5888631343841553, 1.5714274644851685]
  - [1.570817470550537, -2.4434248409666957, 2.4434624353991907, -1.5707813241236472, 1.570798635482788, 1.6919137124205008e-05]
  - [2.8729708194732666, -2.0448461971678675, 2.021271053944723, -1.5182170134833832, 1.5593169927597046, -1.2972553412066858]
  - [3.101041793823242, -1.3452726167491456, 1.4374216238604944, -1.6605121097960414, 1.5967109203338623, -1.5743745009051722]
  velocities:
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  time_from_start:
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  joint_names:
  - shoulder_pan_joint
  - shoulder_lift_joint
  - elbow_joint
  - wrist_1_joint
  - wrist_2_joint
  - wrist_3_joint

from_4_to_home:
  positions:
  - [3.101041793823242, -1.3452726167491456, 1.4374216238604944, -1.6605121097960414, 1.5967109203338623, -1.5743745009051722]
  - [3.061617851257324, -1.627420564691061, 2.1322463194476526, -2.0675469837584437, 1.6000657081604004, -1.5212863127337855]
  - [2.846527576446533, -2.3439976177611292, 2.5685952345477503, -1.7792545757689417, 1.5839970111846924, -1.2860940138446253]
  - [2.0300660133361816, -2.496726175347799, 2.3432703653918665, -1.3323903244784852, 1.5964887142181396, -0.5053618589984339]
  - [1.5707390308380127, -2.443453451196188, 2.443479601536886, -1.5708095035948695, 1.5707736015319824, 9.5367431640625e-07]
  velocities:
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  time_from_start:
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  joint_names:
  - shoulder_pan_joint
  - shoulder_lift_joint
  - elbow_joint
  - wrist_1_joint
  - wrist_2_joint
  - wrist_3_joint